ios/AppLovinMAX.m

```diff
@@ -2266,12 +2267,33 @@ - (void)setAmazonResult:(id /* DTBAdResponse or DTBAdErrorInfo */)result forAdUn
     if ( adFormat == MAAdFormat.interstitial )
     {
         MAInterstitialAd *interstitial = [self retrieveInterstitialForAdUnitIdentifier: adUnitIdentifier];
-        [interstitial setLocalExtraParameterForKey: key value: result];
+        
+        if (interstitial)
```

Comment:
Formatting and can we keep code flow consistency with Android?
```
 MaxInterstitialAd interstitial = retrieveInterstitial( adUnitId, "setAmazonResult" );
            if ( interstitial == null )
            {
                e( "Unable to set Amazon result - unable to retrieve interstitial" );
                return;
            }
```

both platforms code should look the same